acceptance/acceptance_test.go

```diff
@@ -361,6 +363,7 @@ func runTest(t *testing.T, dir, coverDir string, repls testdiff.ReplacementsCont
 	absDir, err := filepath.Abs(dir)
 	require.NoError(t, err)
 	cmd.Env = append(cmd.Env, "TESTDIR="+absDir)
+	cmd.Env = append(cmd.Env, "CLOUD_ENV="+cloudEnv)
```

Comment:
This is not necessary; the command already inherits the parent's environment, so `CLOUD_ENV` is already set.

Reply:
Local acceptance tests are supposed to be run with empty environment variables. If we do not set this variable here, then I get an error in the script when trying to check for CLOUD_ENV (as a marker that this test is running locally or in the cloud): `CLOUD_ENV: unbound variable`. I believe possible workarounds include modifying bash error levels with `set -u` or `set -o nounset`, but I'd rather have this line here than managing error levels in bash for this specific purpose. Please re-open if you think there is a better way of handling this.